implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -185,6 +198,20 @@ void poll() {
 
     public void start() {
         if (started.compareAndSet(false, true)) {
+            if (statsdConfig.enabled()) {
+                Sinks.Many<String> sink = Sinks.unsafe()
+                                               .many()
+                                               .unicast()
+                                               .onBackpressureBuffer(new MpscUnboundedArrayQueue<>(256));
```

Comment:
this should resolve problems with high CPU contention

Reply:
Alternatively, we can have a bounded Multiproducer queue here since as far as I understand we can drop part of the values